src/core/biblio.js

```diff
@@ -93,24 +93,152 @@ export function wireReference(rawRef, target = "_blank") {
   `;
 }
 
-export function stringifyReference(ref) {
+// const html = (...args) => hyperHTML.bind(doc.createDocumentFragment())(...args);
```

Comment:
We should probably get rid of this commented out code.

Reply:
Ah, sorry... this is the updated code... can you uncomment this, and remove the other stuff?